packages/composer-popover/components/ComposerWrapper/index.jsx

```diff
@@ -17,7 +17,11 @@ const ConnectedComposer = props => (
   </Suspense>
 );
 
-const getProfiles = (state, selectedProfileId) => {
+const getProfiles = (
+  state,
+  selectedProfileId, // Currently selected and open profile ID
```

Comment:
I believe the descriptions you added in the interfaces file are easier to read 🙂 
```suggestion
  selectedProfileId, // Provided when in the context of a single profile (e.g. queue)
```

Reply:
Ah great, will update it then 😄 Glad at least one of the descriptions are useful!